### DISCUSSION TITLE
Meaning of binding_energy	in result table

### DISCUSSION THREAD

**Discussion by jansteinkuehler** [KosinskiLab/AlphaPulldown#403] [Q&A] [answered]
Hello,
I could not find an explanation in the manual or AlphaPulldown paper what the value of binding_energy that is generated by alpha-analysis_jax_0.4.sif is / how it is calculated.

Could you point me where I can find further information?

Thanks!
Jan

**Comment by DimaMolod** [comment, marked as answer]
Hi @jansteinkuehler!
I think 'binding energy' here is calculated as 'energy' of complex minus 'energy' of individual monomers, whereas 'energy' is calculated by `pyrosetta.rosetta.core.scoring.get_score_function`
https://github.com/KosinskiLab/AlphaPulldown/blob/main/alphapulldown/analysis_pipeline/pdb_analyser.py#L225
you are right and we should update the manual @gilep
